core/iwasm/aot/aot_runtime.h

```diff
@@ -35,6 +35,7 @@ extern "C" {
  * and not at the beginning of each function call */
 #define WASM_FEATURE_FRAME_PER_FUNCTION (1 << 12)
 #define WASM_FEATURE_FRAME_NO_FUNC_IDX (1 << 13)
+#define WASM_FEATURE_BRANCH_HINTS (1 << 14)
```

Comment:
do we need this flag?
branch hinting doesn't produce any incompatibility in the aot file, does it?